tools/keepalived/keepalived/check/ipvswrapper.c

```diff
@@ -364,10 +364,12 @@ ipvs_change_needed(int cmd, virtual_server_group_entry_t *vsge, virtual_server_t
 
 		return (count == 0);
 	}
+#if 0
```

Comment:
Did nothing for this commit？？

Reply:
The codes are controversial. Keep the original codes for references of future debug.